apprentice_teacher.rb

```diff
@@ -1,23 +1,20 @@
-class ApprenticeTeacher
+require_relative 'student'
+require_relative 'highfiveable'
+require_relative 'teacher_do'
+
+class ApprenticeTeacher < Student
+  include HighFiveable
```

Comment:
This reads as "an apprentice teacher is a kind of student" which doesn't sound quite right. How would you fix this?